packages/website/src/components/Testimonials/Testimonials.data.ts

```diff
@@ -48,7 +48,7 @@ export const testimonials: Testimonial[] = [
   },
   {
     image: 'https://randomuser.me/api/portraits/women/88.jpg',
-    name: 'Darlene Mccoy',
+    name: 'Darlene Mccoy-wahad',
```

Comment:
🙃 Is it a planned change?

Reply:
xd 
When i copied data, I didn't change that name and this is the key prop... so, frustrating warning in console appeared